src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const partnerType = (person.sex === 'm') ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+      {person.age && <p className="Person__age">{`I am ${person.age}`}</p>}
+
+      <p className="Person__partner">
+        {person.isMarried ? (
+          `${person.partnerName} is my ${partnerType}`
+        ) : (
+          'I am not married'
+        )}
```

Comment:
```suggestion
        {person.isMarried
          ? `${person.partnerName} is my ${partnerType}`
          : 'I am not married'
        }
```

Reply:
But the syntax I used is taken from the React docs and was also used by Misha.